### PR TITLE
Add docker folder with Dockerfile and readme

### PR DESCRIPTION
**Pull request by NickKok**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+WORKDIR /workspace
+RUN chmod -R a+w /workspace
+
+
+
+
+
+
+
```
Comment (ManosMagnus) [inline docker/Dockerfile:64]: Please remove empty lines

File: docker/preprocessors.py [hunk 2]
```diff
@@ -0,0 +1,357 @@
+from collections import OrderedDict
```
Comment (ManosMagnus) [inline docker/preprocessors.py:1]: We need to add the original source of this code file

File: docker/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+gym==0.17.2
+numpy==1.19.2
+tensorflow==2.5.0
+stable-baselines3
+ray==1.2.0
+torch==1.8.1
+wandb
+
```
Comment (ManosMagnus) [inline docker/requirements.txt:8]: Remove empty line

**Review by ManosMagnus — CHANGES_REQUESTED** [review]
I've added some minor comments. 

Please squash on one commit in order to method them when you address the mentioned comments.

Thank you

File: docker/preprocessors.py (hunk 2 above)
Comment (NickKok) [reply docker/preprocessors.py:1]: The original code file doesn't work when you are using the Ray with customer environment. I could verify it again but I remember that I had issues using the original file.

Comment (ManosMagnus) [reply docker/preprocessors.py:1]: I mean just the source (a link or something) for future reference

Comment (NickKok) [reply docker/preprocessors.py:1]: Ok, should I add it as a comment on the docker file or on the preprocessor .py file?

Comment (ManosMagnus) [reply docker/preprocessors.py:1]: In preprocessor file it's fine

**Comment by ManosMagnus** [comment]
@all-contributors please add @NickKok for code, doc, idea

**Comment by ManosMagnus** [comment]
Thank you!

**Pull request merged**